runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -783,17 +783,45 @@ void J9::ARM64::PrivateLinkage::createPrologue(TR::Instruction *cursor)
 
       int32_t preservedRegisterOffsetFromJavaSP = outgoingArgsSize;
 
+      TR::RealRegister *regToStore = NULL;
```

Comment:
Minor nit: can you rename this to `pendingRegToStore` for improved readability?

Reply:
Changed as suggested.